test/test_routes.py

```diff
@@ -39,4 +39,20 @@ def client():
 
 def test_latest_article(client):
     response = client.get("/latest-article")
-    assert response.json == {"response": "OK"}
\ No newline at end of file
+    assert response.json == {"response": "OK"}
```

Comment:
This is a weak test. The function can return anything with a 200 status and pass this test.